### PR TITLE
Add ruff-lsp

### PR DESCRIPTION
**Pull request by berceanu**
Add recipe for ruff language server protocol

@conda-forge/help-python, ready for review

### CODE REVIEW

File: recipes/ruff-lsp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  commands:
+    - pip check
+    - ruff-lsp --help
+  requires:
+    - pip
+
+about:
+  summary: A Language Server Protocol implementation for Ruff.
+  license: MIT
+  license_file: LICENSE
```
Comment (dhirschfeld) [inline recipes/ruff-lsp/meta.yaml:44]: The linter is complaining that this is missing a required `home` key - e.g.
https://github.com/conda-forge/staged-recipes/blob/7b820acf2a70a2b92040b4d7929aff7df5f0b9aa/recipes/example/meta.yaml#L66-L67
<br>
Not sure why `grayskull` didn't add that, but the below should do the trick:
```suggestion
about:
  home: https://github.com/astral-sh/ruff-lsp
  summary: A Language Server Protocol implementation for Ruff.
  license: MIT
  license_file: LICENSE
```

File: recipes/ruff-lsp/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  imports:
+    - ruff_lsp
+  commands:
+    - pip check
+    - ruff-lsp --help
+  requires:
+    - pip
+
+about:
+  summary: A Language Server Protocol implementation for Ruff.
```
Comment (ruben-arts) [inline recipes/ruff-lsp/meta.yaml:42]:
```suggestion
  summary: A Language Server Protocol implementation for Ruff.
  home: https://github.com/astral-sh/ruff-lsp
```

File: recipes/ruff-lsp/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+about:
+  home: https://github.com/astral-sh/ruff-lsp
+  summary: A Language Server Protocol implementation for Ruff.
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - berceanu
```
Comment (berceanu) [inline recipes/ruff-lsp/meta.yaml:49]: @moritzwilksch

Comment (moritzwilksch) [inline recipes/ruff-lsp/meta.yaml:49]:
```suggestion
    - berceanu
    - moritzwilksch
```

Comment (zanieb) [reply recipes/ruff-lsp/meta.yaml:49]:
```suggestion
    - berceanu
    - moritzwilksch
    - zanieb
```

**Comment by moritzwilksch** [comment]
@conda-forge-admin, please ping conda-forge/help-python

**Pull request merged**